src/webui/www/private/scripts/client.js

```diff
@@ -95,7 +95,51 @@ const getShowFiltersSidebar = function() {
     return (show === null) || (show === 'true');
 };
 
+function isValidIpv4Address(ip) {
+    // Regular expression to validate IPv4 address format
+    const ipv4Regex = /^(?:[0-9]{1,3}\.){3}[0-9]{1,3}$/;
+    return ipv4Regex.test(ip);
+}
+
+function isValidIpv6Address(ip) {
+    // Regular expression to validate IPv6 address format
+    const ipv6Regex = /^(?:[A-F0-9]{1,4}:){7}[A-F0-9]{1,4}$/i;
```

Comment:
This won't work with compressed forms, for example: `2001:db8:0:0:1::1`.
Maybe stackoverflow would have some solutions.

Reply:
resolve via https://github.com/qbittorrent/qBittorrent/pull/18190/commits/8feb85835ad85ddee4ada726ae431d2d757f15cd